src/components/hmi_message_handler/src/messagebroker_adapter.cc

```diff
@@ -60,10 +61,12 @@ void MessageBrokerAdapter::SendMessageToHMI(
     return;
   }
 
-  Json::Reader reader;
+  JsonReader reader;
   Json::Value json_value;
-  if (!reader.parse(message->json_message(), json_value, false)) {
-    LOG4CXX_ERROR(logger_, "Received invalid json string.");
+  std::string str = message->json_message();
```

Comment:
@ydementieiev const?

Reply:
@AByzhynar done in [1345b2e](https://github.com/smartdevicelink/sdl_core/pull/3155/commits/1345b2e4f52415e896a5fa3d232aa391c69751bb)